example-app/src/main/java/backtraceio/backtraceio/MainActivity.java

```diff
@@ -184,4 +190,16 @@ private void writeMyCustomFile(String filePath) {
     public void exit(View view) {
         System.exit(0);
     }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        backtraceClient.onResume();
```

Comment:
Requiring the user to do this will make `BacktraceClient` much more complicated to use for the customer. There's probably a way for `BacktraceClient` to capture these signals without the user having to do something, using [ActivityLifecycleCallbacks ](https://developer.android.com/reference/android/app/Application.ActivityLifecycleCallbacks) for instance?